### ISSUE TITLE
Keyboard navigation

### ISSUE THREAD

**Issue by matthewp** [matthewp/comic-reader#30]
This broke when I did the pinch to zoom feature.

**Comment by matthewp** [comment]
Stuff that works:

* left/right to navigation pages

Stuff that doesn't work

* [ ] cmd+Up to zoom in
* [ ] up/down to scroll
* [ ] left/right to pan when zoomed in
